Signal/src/ViewControllers/NewContactThreadViewController.m

```diff
@@ -823,11 +823,8 @@ - (void)newConversationWithRecipientId:(NSString *)recipientId
 - (void)newConversationWithThread:(TSThread *)thread
 {
     OWSAssert(thread != nil);
-    [self dismissViewControllerAnimated:YES
-                             completion:^() {
-                                 [SignalApp.sharedApp presentConversationForThread:thread
-                                                                            action:ConversationViewActionCompose];
-                             }];
+    [SignalApp.sharedApp withoutAnimationPresentConversationForThread:thread action:ConversationViewActionCompose];
+    [[self presentingViewController] dismissViewControllerAnimated:YES completion:nil];
```

Comment:
Is there an analogous change we could make in the "new group thread" view?  I'm assuming no change is need in the "home view search" view?